docker-compose.yml

```diff
@@ -3,10 +3,14 @@ version: '2'
 services:
   studio:
     build: .
-    command: gulp
+    command: yarn run start
     ports:
-      - "3000:3000"
-      - "3001:3001"
+      - "8080:8080"
```

Comment:
maybe it could be 3006, and give a domain name in:  https://gist.github.com/santiagodoldan/e1e0108f6ff099a6929729fb7610032f